meta-oe/recipes-extended/libbacktrace/libbacktrace_git.bb

```diff
@@ -23,5 +23,7 @@ do_configure() {
     oe_runconf
 }
 
+ALLOW_EMPTY_${PN} = "1"
```

Comment:
Instead try if the following works. We dont want empty packages if possible.
```
RDEPENDS:${PN}-dev:remove = "${PN}"
```

Reply:
unfortunately, your suggestion ends up in a python error 

```
...                                                                                                                                                    
ERROR: libbacktrace-1.0+gitAUTOINC+4f57c99971-r0 do_package: Error executing a python function in exec_func_python() autogenerated:                                          
                                                                                                                                                                             
The stack trace of python calls that resulted in this exception/failure was:                                                                                                 
File: 'exec_func_python() autogenerated', lineno: 2, function: <module>                                                                                                      
     0001:                                                                                                                                                                   
 *** 0002:read_shlibdeps(d)                                                                                                                                                  
     0003:                                                                                                                                                                   
File: '/workspace/build/../poky/meta/classes/package.bbclass', lineno: 2242, function: read_shlibdeps                   
     2238:                rdepends[dep] = []                                                                                                                                 
     2239:            for v in pkglibdeps[pkg][dep]:                                                                                                                         
     2240:                if v not in rdepends[dep]:                                                                                                                         
     2241:                    rdepends[dep].append(v)                                                                                                                        
 *** 2242:        d.setVar('RDEPENDS:' + pkg, bb.utils.join_deps(rdepends, commasep=False))                                                                                  
     2243:}                                                                                                                                                                  
     2244:                                                                                                                                                                   
     2245:python package_depchains() {                                                                                                                                       
     2246:    """                                                                                                                                                            
File: '/workspace/poky/bitbake/lib/bb/utils.py', lineno: 278, function: join_deps                                       
     0274:    for dep in deps:                                                                                                                                               
     0275:        if deps[dep]:                                                                                                                                              
     0276:            if isinstance(deps[dep], list):                                                                                                                        
     0277:                for v in deps[dep]:                                                                                                                                
 *** 0278:                    result.append(dep + " (" + v + ")")                                                                                                            
     0279:            else:                                                                                                                                                  
     0280:                result.append(dep + " (" + deps[dep] + ")")                                                                                                        
     0281:        else:                                                                                                                                                      
     0282:            result.append(dep)                                                                                                                                     
Exception: TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'                                                                                                
                                                                                                                                                                             
ERROR: Logfile of failure stored in: /work/cortexa53-crypto-oe-linux/libbacktrace/1.0+gitAUTOINC+4f57c99971-r0/temp/log.do
_package.147652                                                                                                                                                              
ERROR: Task (/workspace/build/../meta-openembedded/meta-oe/recipes-extended/libbacktrace/libbacktrace_git.bb:do_package)
 failed with exit code '1'                                                                                                                                                   
NOTE: Tasks Summary: Attempted 5097 tasks of which 5089 didn't need to be rerun and 1 failed.                                                                                
                                                                                                                                                                             
Summary: 1 task failed:                                                                                                                                                      
  //workspace/build/../meta-openembedded/meta-oe/recipes-extended/libbacktrace/libbacktrace_git.bb:do_package            
Summary: There was 1 WARNING message.                                                                                                                                        
Summary: There was 1 ERROR message, returning a non-zero exit code.                                                                                                          
```

Reply:
try
`RDEPENDS:${PN}-dev = ""`

Reply:
yes, this works. I just updated the PR.